NineChronicles.Headless.Executable.Tests/Commands/ActionCommandTest.cs

```diff
@@ -182,7 +182,7 @@ public void ClaimStakeReward(string addressString, int expectedCode)
             }
             else
             {
-                Assert.Contains("System.FormatException: Input string was not in a correct format.", _console.Error.ToString());
+                Assert.Contains("System.FormatException: The input string '0x' was not in a correct format", _console.Error.ToString());
```

Comment:
If this pull request must be merged asap, it should be separated into another commit (if upgrading .NET 7 doesn't break the build)